solidity/migrations/2_deploy_contracts.js

```diff
@@ -57,6 +58,11 @@ const all = [
 
 module.exports = (deployer, network, accounts) => {
   deployer.then(async () => {
+    if (network == "keep_dev" || network == "local") {
```

Comment:
Hmmm... I think this is a difference between here and one of our other projects. I’ll either rename that one local for consistency or change this to refer to development.

Reply:
We do indeed use `local` in keep-ecdsa and keep-core both, but rather than disentangle this against truffle's preference for `development` as the default network I'm going to just tweak this conditional. More later.